### DISCUSSION TITLE
Strange training/validation results

### DISCUSSION THREAD

**Discussion by jramborger78** [talmolab/sleap#1593] [Help!]
Hi all! I have never seen this before, can anyone possibly enlighten me on if this is a problem or what is going on here? It regularly is triangulating, I have never seen a validation go backwards and put the point down at an earlier batch point.
![q](https://github.com/talmolab/sleap/assets/134438857/9c081190-2eee-4cf8-b8f0-4f09755bce15)

Also, I have seen a couple of my frames will have 2, even 3, predications on it. When I correct those should I correct them all? One? Delete the ones I don't correct? Thanks!

**Comment by talmo** [comment]
Whew, this looks like a doozy!! We definitely should NOT be seeing these lines running in parallel to each other.

I'm suspecting that this is happening because you have two training jobs running at the same time. This can happen sometimes if the GUI crashes during a training run, but the training subprocess keeps executing. Normally this would just result in a GPU out-of-memory error, but this might be a particularly light model (or you have lots of GPU memory).

I'd recommend killing all `python.exe` and `sleap-train.exe` processes running in the background through your system Task Manager, or just restarting the computer.

---

> Also, I have seen a couple of my frames will have 2, even 3, predications on it. When I correct those should I correct them all? One? Delete the ones I don't correct? Thanks!

If they're predicted instances (colored yellow and you can't move them), then you can just ignore them. If they're user instances (you can move the nodes), then they need to be correct and you can't have any extras.

As you improve the model you should see fewer false positives extra instances.

**Comment by jramborger78** [comment]
That is very interesting and will keep that in mind if I see it occur again! Glad to know I wasn't crazy. Thank you, sir.
